harness/php/src/Feature/WorkflowStubInjector.php

```diff
@@ -51,6 +52,7 @@ public function createInjection(
         $feature = $this->container->get(Feature::class);
         $options = WorkflowOptions::new()
             ->withTaskQueue($feature->taskQueue)
+            ->withWorkflowIdReusePolicy(IdReusePolicy::AllowDuplicate)
```

Comment:
Looks like PHP SDK uses `AllowDuplicateFailedOnly` by default there.
Is it OK?

Reply:
Either way I don't think you could change the default for compatibility reasons, so I'd say it's fine.

Reply:
But we can add a feature flag and change the behavior in the future